modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json

```diff
@@ -7,7 +7,7 @@
 		"ckeditor4-react": "1.4.2",
 		"ckeditor5": "43.3.1",
 		"frontend-js-web": "*",
-		"liferay-ckeditor": "4.21.0-liferay.5",
+		"liferay-ckeditor": "4.21.0-liferay.9",
```

Comment:
Sorry, because that was all my bad.

 * I released `v4.21.0-liferay.6` with a possible solution, however after releasing I realized it was fixing this case, but at the same time, causing a regression on a different part.
 * So then I released `v4.21.0-liferay.7` reverting previous tries to set a default width to images since I realized it was better to fix it inside `liferay-portal`. For some reason I don't know (probably I did something wrong in the release process) `ckeditor.js` was not updated reverting changes.
 * Then I redo the process running `./ck.sh build ` and releasing a `v4.21.0-liferay.8`, checking changes were actually reverted as I expected, but after releasing I realised tests in `liferay-portal` started failing 😱 . `ckeditor.js` was quite different from previous version, so even when it was funcionally and aparently similar, some locators were not found in playwright.
 * Then I realized even when I just wanted to release a new version with no changes I had to go through the whole process `./ck.sh setup && ./ck.sh patch && ./ck.sh build` and after doing that I published `v4.21.0-liferay.9` which properly reverts my changes and doesn't modify the editor in a way that makes playwright test to fail.

😞 

About those houskeeping nits, I've just done it. Thanks for letting me know, I didn't remember there were some more steps in the publishing process after running `yarn publish`. I'll keep it in mind next time 🙌